README.md

```diff
@@ -24,10 +24,30 @@ represents one thing that your application *does*.
 
 ### Context
 
-An interactor is given a *context*. The context contains everything the
-interactor needs to do its work.
+An interactor is given a *context*.  The context contains everything the
+interactor needs to do it's work.
 
-When an interactor does its single purpose, it affects its given context.
+The context is created when the interactor is called with a hash:
+
+In the example below the hash { email: 'example@example.com'} is passed 
+and is turned into OpenStruct object and becomes the context.
```

Comment:
I feel that "OpenStruct object.." and "defines a class method..." (below) are details you find if you dig through the code, not a README.  If the context is not clear, maybe we update the existing example code to be more clear?

Thoughts @laserlemon @ersatzryan?